corehq/apps/builds/management/commands/add_commcare_build.py

```diff
@@ -1,26 +1,91 @@
-from django.core.management.base import BaseCommand, CommandError
+import random
+from github import Github
 
-from corehq.apps.builds.models import CommCareBuild
+from django.core.management.base import BaseCommand, CommandError
+from corehq.apps.builds.models import CommCareBuild, CommCareBuildConfig, BuildMenuItem, BuildSpec
 
 
 class Command(BaseCommand):
     help = ('Adds a commcare build, labeled with the version (x.y.z) and build_number (an incrementing integer)\n'
             'to get started see https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/builds/README.md')
 
     def add_arguments(self, parser):
-        parser.add_argument('build_path')
-        parser.add_argument('version')
-        parser.add_argument('build_number')
+        parser.add_argument('build_path', nargs='?')
+        parser.add_argument('version', nargs='?')
+        parser.add_argument('build_number', nargs='?')
+        parser.add_argument(
+            '-l',
+            '--latest',
+            action='store_true',
+            help="add the latest remote CommCare build version"
```

Comment:
"Remote" has a couple of meanings in this codebase. How about being explicit that this will get the last version from GitHub?